### PR TITLE
feat(handler): add error validation logic and toast for de minimis aid form (HL-1202)

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

Add validation for handler's de minimis form when creating a new application.

### CODE REVIEW

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2933.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2933.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2933.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2933.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2933.dev.hel.ninja :laughing::tada::tada::tada:

File: frontend/benefit/handler/public/locales/en/common.json
Comment (EmiliaMakelaVincit) [inline frontend/benefit/handler/public/locales/en/common.json:341]: -  'Lisää' painiketta -> Lisää-painiketta / 'Lisää'-painiketta
- minimis -kentät -> de minimis -kentät

File: frontend/benefit/handler/src/components/applicationForm/formContent/companySection/deMinimisAid/DeMinimisAidForm.tsx
Comment (EmiliaMakelaVincit) [inline frontend/benefit/handler/src/components/applicationForm/formContent/companySection/deMinimisAid/DeMinimisAidForm.tsx:67]: This effectively just equals to `setUnfinishedDeMinimisAidRow(isFormInputIncomplete())`, the if-else is redundant.

**Review by EmiliaMakelaVincit — APPROVED** [review]
Some nitpicks, can be merged without another review after possible fixes.

**Pull request merged**
